frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -169,6 +171,12 @@ private void refreshModel(GitRepository gitRepository) {
       LOG.info("Machete file (${macheteFilePath}) is absent");
     }
 
+    if (!notCreatedBranchNames.isEmpty()) {
+      // This warning notification will not cover other error notifications (e.g. when rebase errors occur)
+      VcsNotifier.getInstance(project).notifyWarning("Some of branches does not exist as a local branch, so was omitted",
```

Comment:
`The following branches defined by machete file do not belong to the local repository:` :question: 

Or even to be perfectly accurate we can distinguish singular and plural case. :microscope: